src/blackboxTests/src/test/kotlin/org/vaccineimpact/api/blackboxTests/schemas/CSVSchema.kt

```diff
@@ -12,7 +12,7 @@ class CSVSchema(schemaFileName: String)
 {
     val columns = parseSchema(schemaFileName)
 
-    fun validate(csvAsString: String)
+    fun validate(csvAsString: String): Iterable<IndexedValue<Array<String>>> 
```

Comment:
Do you need this to be indexed? In `coverage sets are indexed from one` you invoke `csvRows.first()`, which I think you can do without having applied indexing.